### ISSUE TITLE
Lens doesn't have recipie

### ISSUE THREAD

**Issue by Kvarts1** [McJty/DeepResonance#289]
I'm playing a modpack for 1.18.1(https://www.curseforge.com/minecraft/modpacks/tnp-limitless-5) and the lens used with the laser doesn't have a recipe.

**Comment by Jkosio23** [comment]
Also an issue with ATM 7 which is currently (0.2.45) using version 3.0.2-beta

**Comment by McJty** [comment]
yes I know. Will be fixed later. For now you can easily add the recipe yourselves using a small datapack

**Comment by bunnyman14** [comment]
Good to know I'm not crazy. I'll just cheat it in since I don't want to bother searching the entire internet for a niche data pack.

**Comment by McJty** [comment]
Oops, forgot about this. Let me fix this for the next version
